irohad/ametsuchi/wsv_command.hpp

```diff
@@ -42,32 +56,35 @@ namespace iroha {
        * @param role_name
        * @return true if insert successful
```

Comment:
Outdated return doc, fix the following as well plz

Reply:
Sure, will fix